src/main/java/com/google/sps/data/GoodDeed.java

```diff
@@ -23,21 +23,26 @@
     private final String description;
     private boolean posted_yet;
     private final long timestamp;
+    private final String link;
  
-    public GoodDeed (Key key, long id, String title, String description, boolean posted_yet, long timestamp) {
+    public GoodDeed (Key key, long id, String title, String description, boolean posted_yet, long timestamp, String link) {
```

Comment:
if you find this getting longer, consider adding a `Builder`. There will be less room for error when setting the attributes.

Reply:
In Kotlin, you have named arguments, but alas, Java doesn't make it easy on us